massdriver-application-azure-app-service/dns.tf

```diff
@@ -1,6 +1,10 @@
+locals {
+  zone_name = element(split("/", var.dns.zone_id), length(var.dns.zone_id) - 2)
```

Comment:
This is close but `length` of the string `var.dns.zone_id` won't get us the right value here. I think you wanted length of the split array.

Reply:
This is what I get when testing in console: 
![image](https://user-images.githubusercontent.com/8037512/191072294-2bca77a2-d235-448c-acdb-a974f824fe96.png)

Reply:
Yes, but I think this is what's going on. 

```terraform
# you're getting the length of the string
length("/one/two/three/four")
19

# when the index passed to element is greater than the number of elements in the array
# it returns the last one
element(["one", "two", "three"], 17)
"three"
```